lapce-ui/src/picker.rs

```diff
@@ -705,6 +705,137 @@ impl Widget<LapceTabData> for FilePickerExplorer {
     }
 }
 
+#[allow(clippy::too_many_arguments)]
+pub fn paint_file_node_item_by_index(
```

Comment:
What even is FilePicker?

Reply:
That's the file dialog to open folder when you connect to remote.